src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var React = require('React');
+var ReactDOM = require('ReactDOM');
+var ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
+
+describe('ReactDOMFiber', () => {
+  var container;
+
+  beforeEach(() => {
+    container = document.createElement('div');
+  });
+
+  it('should render a component returning strings directly from render', () => {
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
@sebmarkbage Thanks! I've updated it. But In case of `ReactDOMFeatureFlags.useFiber` is false, this test is failed because of `Your test suite must contain at least one test.`.
So I added some tests that can pass regardless the fiber flag.
Does this make sense?